server/controllers/customers/customer-controller.js

```diff
@@ -0,0 +1,44 @@
+const models = require('../../models')
+const { sendTwilioSMS } = require('../../utilities/send-sms')
+
+// Get all Cusotmers
+const getCustomers = (req, res) => {
+  models.Customer.findAll()
+    .then(data => res.send(data))
+    .catch(err => {
+      res.status(500).send(err.message || "Some error occured while retrieving customers")
+    })
+}
+
+// Send Message
+const sendMessage = async (req, res) => {
+  try {
+    const { id } = req.params
+    const { message } = req.body
```

Comment:
Do you see here? the route should be the one responsible for doing this.

Reply:
Same here - are you saying that the controller function should not be handling the request value in this way?

Reply:
Here the route should strip the request, and pass to send message id, and message. Something like this
controller
```
const sendMessage = async (id, message) => { ... }
```
route
```
(req, res) => {
  const {id} = req.params
  const {message} = req.body
  sendMessage(id, message).then().catch()
}
````